### DISCUSSION TITLE
Importing 'flutter' material is misleading?

### DISCUSSION THREAD

**Discussion by a-wallen** [ethanblake4/flutter_eval#57] [Ideas]
The examples suggest that we import material in the dynamically evaluated code, yet not all classes from flutter are implemented. I think it would be worth it to show only the classes that are implemented so that a plugin developer knows what is available.

**Comment by ethanblake4** [comment]
Thanks for the feedback. Since this project does aim to eventually cover all flutter classes, importing flutter is done to reduce the amount of changes developers will have to make to their code. Additionally I have limited bandwidth to work on this project and I do not feel that creating a complete copy of all the Flutter classes as interfaces for devs to import is a good use of my time.
